projects/angular/src/forms/styles/_properties.forms.scss

```diff
@@ -12,29 +12,38 @@
   @if variables.$clr-use-custom-properties == true {
     :root {
       // Structure
+      --clr-forms-baseline: #{variables.$clr_baselineRem_0_25};
       --clr-form-disabled-background-color: var(--clr-color-neutral-400);
 
       // Generic
       --clr-forms-label-color: var(--clr-color-neutral-800);
       --clr-forms-text-color: var(--clr-color-neutral-1000);
       --clr-forms-invalid-color: var(--clr-color-danger-800);
+      --clr-forms-invalid-text-color: var(--clr-color-danger-800);
       --clr-forms-valid-color: var(--clr-color-success-700);
       --clr-forms-valid-text-color: var(--clr-color-success-900);
       --clr-forms-subtext-color: var(--clr-color-neutral-600);
       --clr-forms-placeholder-color: var(--clr-global-color-construction-600);
       --clr-forms-border-color: #{forms-variables.$clr-forms-border-color};
-      --clr-forms-focused-color: var(--clr-color-action-600);
+      --clr-forms-focused-color: var(--clr-color-action-500);
```

Comment:
Why is that change needed. It will affect only clarity theme.

Reply:
Yeah it was not needed, I was just looking at something when I changed it